src/README.md

```diff
@@ -45,3 +48,21 @@ If you want to change the domain the shop is available at, you have to update it
 
 > [!CAUTION]
 > This method is not ready - we need a method to save the database and the files in the repository.
```

Comment:
Can we remove this disclaimer - the issue seems to be solved?

Reply:
Sure, will remove it